src/Extensions/ElementalSubsitesPageExtension.php

```diff
@@ -45,4 +48,24 @@ public function onAfterDuplicateToSubsite($originalPage)
             );
         }
     }
+
+    /**
+     * Extension hook {@see ElementalAreasExtension::requireDefaultRecords}
+     *
+     * Sets Subsite::$disable_subsite_filter to true, reruns
+     * ElementalAreasExtension's requireDefaultRecords() then
+     * Sets Subsite::$disable_subsite_filter to it's original state
+     *
+     * @return void
+     */
+    public function requireDefaultRecords()
+    {
+        $original = Subsite::$disable_subsite_filter;
+        Subsite::$disable_subsite_filter = true;
+        $instance = Injector::inst()->get(ElementalAreasExtension::class);
```

Comment:
Wouldn't this logic mean that `ElementalAreasExtension::requireDefaultRecords()` ends up running twice?

Reply:
(Friendly) ping =)

Reply:
From my understanding this would override that function and run only here.

Reply:
Yeah you're right. Instead i'll use a hook.

Reply:
@robbieaverill I have closed thin in favor of this PR: https://github.com/dnadesign/silverstripe-elemental-subsites/pull/10